src/components/browse/datasetlist/datasetPagination.tsx

```diff
@@ -58,7 +58,7 @@ const DatasetPagination = (props: dataSetPaginationProps) => {
         breakClassName={"page-item"}
         breakLinkClassName={"page-link"}
         activeClassName={"active"}
-        forcePage={props.currentPage}
+        forcePage={props.page === 0? props.page: props.page -1}
```

Comment:
Why do we do this? If the page is 0, then we use 0, but if it is 1 we also use 0. Same issue as before.

Reply:
The reason we do this because the page can be 0 or positive integer.
the page numbers for the react-paginate is one less than the page selected. In the onClick method for this component, we navigate to the current page + 1.

Reply:
Yes, but if props.page === 0 then you can just write 0, no need to pass props.page is what I mean.